Added Apache Pulsar in Action

stjepanjurekovic opened the pull request:
Hi,

Stjepan from Manning here. I thought this title might be a good match for your list. Thank you for considering it.

Best,

JamesLavin commented:
I'm quite embarrassed I never got to this, @stjepanjurekovic.

There was a merge conflict, due to the addition of another book in the same location, so I was unable to merge this, but I've just added the change in a separate commit, which I've just merged in, https://github.com/JamesLavin/my_tech_resources/commit/76bb006cbe7a27ecc9ae27d4798d2edca9c2e2f3.

Cheers,

James

JamesLavin commented:
Forgot to hit "Close." I've added this change in a separate commit due to a merge conflict. Can't merge this for technical reasons, but I've added the resource.

stjepanjurekovic commented:
Thank you, James.

Best,

On Wed, Jun 14, 2023 at 8:27 PM James Lavin ***@***.***>
wrote:

> Forgot to hit "Close." I've added this change in a separate commit due to
> a merge conflict. Can't merge this for technical reasons, but I've added
> the resource.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JamesLavin/my_tech_resources/pull/8#issuecomment-1591783811>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANDIEMI6ZHLLVG2MLMSJY73XLH7CTANCNFSM5CRVOMDQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Stjepan Jureković
Manning Publications - Marketing
***@***.***
